spring-boot/src/main/java/com/people/peoplemanager/model/Person.java

```diff
@@ -0,0 +1,74 @@
+package com.people.peoplemanager.model;
+
+import jakarta.persistence.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Table(name = "PERSON_TB")
+public class Person {
```

Comment:
Será que também poderia ser um `record`?

Reply:
Eu tentei primeiro fazer e não rolou, fiz uma pesquisa rapida e encontrei isso: https://thorben-janssen.com/java-records-hibernate-jpa/#records-cant-be-entities. No caso acredito que a classe Person não terá como se tornar um record certo?

Reply:
é, não dá não